src/modules/collectionCuration/types.ts

```diff
@@ -4,10 +4,13 @@ export enum CurationStatus {
   REJECTED = 'rejected'
 }
 
```

Comment:
I don't want you to have to move everything again, but what do you think of this:
- Encompassing `collectionCuration` inside `/modules/curation` so it looks like:
```
/modules/curation
  collectionCuration/
  itemCuration/
```

- add top level logic in curation:
```
/modules/curation
  types.ts # contains CurationStatus and BaseCuration for example, useful for both
  collectionCuration/
  itemCuration/
```